website/source/docs/enterprise/namespaces/index.html.md

```diff
@@ -13,7 +13,7 @@ can be isolated from each other with the use of Namespaces. Namespaces help redu
 by removing restrictions around uniqueness of resource names across distinct teams, and enable operators 
 to provide self-service through delegation of administrative privileges.
 
-For more information on how to use namespaces with Consul Enterprise please review the following Learn Guides:
+For more information on how to use namespaces with Consul Enterprise please review the following HashiCorp Learn Guides:
 
 - [Register and Discover Services within Namespaces](https://learn.hashicorp.com/consul/namespaces/discovery-namespaces) - Register multiple services within different Namespaces in Consul
```

Comment:
```suggestion
- [Register and Discover Services within Namespaces](https://learn.hashicorp.com/consul/namespaces/discovery-namespaces) - Register multiple services within different Namespaces in Consul.
```

Reply:
Can you also lower case https://github.com/hashicorp/consul/pull/7495/files#diff-a7c1e35dcabd95e1f65d084fe10ad7f4R12 ?